README.md

```diff
@@ -67,6 +67,22 @@ Varnish PID file path. Set to an empty string if you don't want to use a PID fil
 
 Services that will be started at boot and should be running after this role is complete. You might need to add additional services if required, e.g. `varnishncsa` and `varnishlog`. If set to an empty array, no services will be enabled at startup.
 
+We can setup vhost with different backend as following
+
+    varnish_backend:
```

Comment:
Thanks for the change to doing the logic in the template; that looks a lot simpler to maintain!

For the example here, can you put the description of the variables _after_ the code block?

Also, can you give a little more explanation of how this works / what you'd use it for? E.g. more than a sentence or two, so it's more obvious why someone might use multiple backends/vhosts.